README.md

```diff
@@ -4,14 +4,26 @@
 
 You need python 3.7+, poetry, and some local configuration.  If you are using `bash` and have a working `make` then just issue
 
-    $ make
+    $ make environment
```

Comment:
Clearly I should get round to installing make! For now I've tested by just running the equivalent commands directly in the terminal without issue 👍

Reply:
heh, make is so old school now.  I'm getting attacked for it as much as for my continued vim use